analytics/src/include/firebase/analytics.h

```diff
@@ -456,7 +457,17 @@ void LogEvent(const char* name);
 /// array.
 void LogEvent(const char* name, const Parameter* parameters,
               size_t number_of_parameters);
-// clang-format on
+
+/// Initiates on-device conversion measurement given a user email address on iOS
+/// (no-op on Android). On iOS, requires dependency
+/// GoogleAppMeasurementOnDeviceConversion to be linked in, otherwise it is a
+/// no-op.
+/// @param email_address User email address. Include a domain name for all email
+/// addresses
+///   (e.g. gmail.com or hotmail.co.jp). Remove any spaces in between the email
+///   address.
```

Comment:
What does it mean to remove spaces in between email address?

Reply:
Ok, well I wouldn't want to hold up the merge of this waiting or a reply. For now you could mirror the public iOS documentation here, and then update it later in a new PR when you have clarity and it needs to be documented.

https://firebase.google.com/docs/reference/ios/firebaseanalytics/api/reference/Classes/FIRAnalytics#+initiateondeviceconversionmeasurementwithemailaddress:

Reply:
I asked about that part of the comment, the reply was that they agreed it did not make sense and removed it from iOS comment (https://critique.corp.google.com/cl/450085932)
So I removed that part of this comment as well.